ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -770,6 +770,30 @@ FCU_DFMSS_DSP_ZHM_RTM_CHWZC_CHWDT:
   - CHWZC
   - CHWDT
 
+FCU_DFSS_STM_RTC_RHM:
+  description: "CRAC unit cooling and humidification with compressor and alarm monitoring."
+  is_canonical: true
+  implements:
+  - FCU
+  - DFSS
+  - STM
+  - RTC
+  - RHM
+  opt_uses:
+  - compressor_run_time_accumulator
+  - humidifier_run_time_accumulator
+  - heater_run_time_accumulator
+  - overload_fan_alarm
+  - high_pressure_status
+  - low_pressure_status
+  - overload_compressor_alarm
+  - humidifier_failed_alarm
```

Comment:
same. move to the humidifier as optional

Reply:
moved to RHC in opt_uses